updatecli/updatecli.d/manifest.yaml

```
@@ -160,7 +160,8 @@ actions:
     kind: github/pullrequest
     scmid: turtles
     spec:
-      automerge: false
+      automerge: true
       mergemethod: squash
       labels:
-        - capiproviders
+        - area/dependency
```

```suggestion
        - area/update
```
I see we have an `area/update` label https://github.com/rancher/turtles/labels/area%2Fupdate that we can probably use here.

---

Updated, thanks